backlogger.py

```diff
@@ -148,12 +131,72 @@ def check_query(data):
         )
 
 
+def standard(values, average, count):
+    if count < 2:
+        return 0
+
+    accum = 0
+    for v in values:
+        accum = accum + (v - average) * (v - average)
+
+    return accum / (count - 1)
+
+
+def median(values):
+    arr = sorted(values)
+    return arr[int(len(arr) / 2)]
```

Comment:
can you use https://docs.python.org/3/library/statistics.html ?

Reply:
Good idea. Thanks!